<launch>
    <include file="$(find controls)/launch/pid_controls.launch" />
    <node name="maintain_yaw" pkg="controls" type="maintain_yaw.py" respawn="false" output="screen">
        <param name="desired_yaw" type="double" value="3.1" />
    </node>
</launch>
